Core/HLE/sceKernelInterrupt.cpp

```diff
@@ -826,6 +826,44 @@ static u32 sysclib_strncpy(u32 dest, u32 src, u32 size) {
 	return hleLogSuccessX(SCEKERNEL, dest);
 }
 
+static u32 sysclib_strtol(u32 strPtr, u32 endPtrPtr, int base) {	
+	const char* str;
+	str = Memory::GetCharPointer(strPtr);	
+	char* end = nullptr;
+	int result = strtol(str, &end, base);
+	ERROR_LOG(SCEKERNEL, "Untest %d = sysclib_strtol(%c, %08x, %i)",result, str, endPtrPtr, base);
+	if (Memory::IsValidRange(endPtrPtr, 4))
+		Memory::WriteUnchecked_U32(strPtr + (end - str), endPtrPtr);
+	return result;
+}
+
+static u32 sysclib_strchr(u32 src, int c) {
+	const std::string str = Memory::GetCharPointer(src);
+	ERROR_LOG(SCEKERNEL, "Untest sysclib_strchr(%c, %i)", str.c_str(),c);
+	int index = str.find(str,c);
```

Comment:
I think that I translate java wrong of 
int index = string.getString().indexOf(c);
Please help again.

Reply:
This is probably fine, it looks like the normal function of the C function `strchr()`.  This might give a type conversion warning, though, better to use size_t or cast to int.

-[Unknown]

Reply:
If you only use size_t, `< 0` will never happen.  Should compare to npos and return 0 in that case.

-[Unknown]

Reply:
ok,I use  cast to int instead